<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastCheckDialog</class>
 <widget class="QDialog" name="LastCheckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/logo/rainbow.png</normaloff>:/logo/rainbow.png</iconset>
  </property>
  <widget class="QTableWidget" name="tableWidget_lastCheck">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>370</width>
     <height>300</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::SolidLine</enum>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="pushButton_createPassport">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>351</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать
паспорт</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>211</x>
     <y>351</y>
     <width>75</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string>Вернуться и
переделать</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
